pkg/localkube/ready.go

```diff
@@ -27,7 +27,26 @@ import (
 
 type HealthCheck func() bool
 
-func healthCheck(addr string, lk LocalkubeServer) HealthCheck {
+func healthCheckInsecure(addr string, lk LocalkubeServer) HealthCheck {
```

Comment:
Can you refactor this to be more like

```golang
func healthCheck(addr, lk LocalkubeServer) {
   transport := lk.GetTransport()
   client := http.Client{Transport: transport}
   ...
}
```

`lk.GetTransport()` will return a new http.Transport with the TLSconfig set if it is secure serving, otherwise just a http.Transport{} if not.

Reply:
done